src/Nethermind/Nethermind.Facade/Proxy/Models/Simulate/Error.cs

```diff
@@ -1,11 +1,16 @@
 // SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
 // SPDX-License-Identifier: LGPL-3.0-only
 
+using System.Text.Json.Serialization;
+using Nethermind.Evm;
+
 namespace Nethermind.Facade.Proxy.Models.Simulate;
 
 public class Error
 {
     public int Code { get; set; }
-    public string Message { get; set; }
-    public string? Data { get; set; }
+    public string? Message { get; set; }
+    [JsonIgnore]
+    public EvmExceptionType EvmException { get; set; }
+    public byte[]? Data { get; set; }
```

Comment:
Let's separate the ignored ones visually:
```suggestion
    public byte[]? Data { get; set; }
    
    [JsonIgnore]
    public EvmExceptionType EvmException { get; set; }
```